app/views/insured/families/_consumer_brokers_widget.html.erb

```diff
@@ -1,63 +1,85 @@
-<% current_broker_agency = @person&.primary_family&.current_broker_agency %>
-<% writing_agent =  @person&.primary_family&.current_broker_agency&.writing_agent %>
-<div class="panel panel-default module consumer-broker-widget">
-  <div class="panel-heading">
-    <h3 class="panel-title">
-      <strong>
-        <% if current_broker_agency && display_broker_info_for_consumer %>
-          <%= l10n("insured.your_broker_or_assister") %>
-        <% else %>
-          <%= l10n("insured.select_broker_or_assister") %>
-        <% end %>
-      </strong>
+<% if @bs4 %>
+  <% current_broker_agency = @person&.primary_family&.current_broker_agency %>
+  <% writing_agent =  @person&.primary_family&.current_broker_agency&.writing_agent %>
+  <div class="home-widget p-2">
+    <h3 class="d-flex justify-content-between">
+      <%= l10n("insured.broker_or_assister") %>
+      <%= link_to '#', data: { toggle: 'modal', target: h('#read_only_modal_what_is_a_broker_or_assister') } do %>
+        <i class="fa fa-question-circle ml-4 mt-1" aria-hidden="true"></i>
+        <span class="hide"><%= l10n("insured.what_is_a_broker_or_assister") %></span>
+      <% end %>
     </h3>
-  </div>
-  <div class="panel-body">
-    <div class="text-center">
-      <i aria-hidden="true" class="fa fa-user fa-3x"></i>
+    <div class="pr-2 pl-2 pb-2 w-100">
       <% if current_broker_agency && display_broker_info_for_consumer %>
-        <br/>
-        <span class="word-break">
-          <%= current_broker_agency&.broker_agency_profile&.legal_name&.titleize %>
-        </span>
+        <%= h(link_to l10n("insured.go_to_my_expert"), '', class: 'btn w-100 btn-default') %>
       <% else %>
-        <br/>
-          <%= l10n(".no_selection") %>
+        <%= h(link_to l10n("insured.assign_an_expert"), '', class: 'btn w-100 btn-default') %>
```

Comment:
this link should open the "Help Me Sign Up modal", so should be similar to `<a data-target="#help_with_plan_shopping" data-toggle="modal" style="<%= link_style %>" id="help_me_sign_up" tabindex="0" onkeydown="handleButtonKeyDown(event, 'help_me_sign_up')">`

Reply:
https://github.com/ideacrew/enroll/pull/4022/commits/8d2e0ecdcb50420a27185714173553a5e9dc36fd#diff-9dd370264a65828beb242d7bd2412ed3cb0e855ba1d5c54e99b91004dea7c5d3R16